deploy/README.md

```diff
@@ -0,0 +1,309 @@
+- 4 GB free disk space
+- Docker 22.0, or greater
+- Docker Compose 2.10, or greater
+
+#### Domain Names
+
+You need three DNS records that resolve to your machines IP address.
+
+- Wikibase e.g. "wikibase.mydomain.net"
+- QueryService e.g. "query.mydomain.net"
+- QuickStatements e.g. "quickstatements.mydomain.net"
+
+### Initial Setup
+
+#### Download the WBS Deployment Kit
+
+Checkout the files from Github and move to the subdirectory `deploy`.
+
+```sh
+git clone https://github.com/wmde/wikibase-release-pipeline
+# TODO update to the correct TAG/BRANCH
+git checkout wmde.20
+cd wikibase-release-pipeline/deploy
+```
+
+#### Initial Configuration
+
+Make a copy of the [configuration template](./template.env) in `wikibase-release-pipeline/deploy`
+
+```sh
+cp template.env .env
+```
+
+Set usernames, passwords and domain names in your newly created `.env` file
+according to the instructions in the comments.
+
+#### Starting
+
+Run the following command from within `wikibase-release-pipeline/deploy`
+
+```sh
+docker compose up --wait
+```
+
+The first start can take a couple of minutes. Wait for your shell prompt to return.
+
+🎉 Congratulations, your Wikibase Suite instance should be up and running. Web
+interfaces are available via HTTPS (port 443) on the domain names you
+configured for Wikibase, WDQS Frontend and Quickstatements.
+
+#### Stopping
+
+To stop, use 
+
+```sh
+docker compose down
+```
+
+#### Resetting the Configuration
+
+Most values set in `.env` are copied statically into the respective containers
+after the first time you run `docker compose up`.
+
+To reset the configuration but keep your existing data:
+
+1. Make any needed changes to the values in the `.env` file copied from
+   `template.env` above. NOTE: Do not change `DB_*` values unless you are also
+   re-creating the database ([see below](#removing-wikibase-suite-completely-with-all-its-data).
+2. Delete your LocalSettings.php from the ./config directory.
+3. Remove and re-create containers with:
+
+```sh
+docker compose down
+docker compose up --wait
+```
+
+### Advanced Configuration
+On first launch, WBS Deployment Kit will create files in the `./config` directory next to your `.env` file and the `docker-compose.yml` and `template.env`. This is the configuration of your instance. **You own those files.** Be sure to include them in your backups.
+
+#### `config/LocalSettings.php`
+This file is generated by the [MediaWiki installer script](https://www.mediawiki.org/wiki/Manual:Install.php) and augmented by the Wikibase containers `entrypoint.sh` on first launch. Once this file has been generated, you own it. This means you can make changes, you even might need to make changes on major version updates (as describes in MediaWiki UPGRADE notes). In order to ease keeping track of your own adjustments vs settings generated by the Wikibase container, we provide a file called `LocalSettings.override.php`. This file is the preferred location for your adjustments. See below.
```

Comment:
Worth considering making "UPGRADE" and similar be in-document bookmarks. Not sure how well that works on Github, but I think I've seen it used: https://stackoverflow.com/a/7335259

Reply:
Done